### DISCUSSION TITLE
Clarification on filter usage (my example)

### DISCUSSION THREAD

**Discussion by lamyergeier** [FreshRSS/FreshRSS#6699] [Q&A]
# What I want to do?

I want to mark feeds as read if it contains any of the following keywords (sample, **in actual I have around 300 keywords**) 

```
advertisement
Advertisement
Afghan
Afghanistan
```

# What I did?

- Do I need to write it like following in **Filter actions**

```
intitle:'advertisement' OR intitle:'Advertisement' OR intitle:'Afghan' OR intitle:'Afghanistan'
```

# Question

1. Is it case sensitive?
2. Do we need to use **OR** instead of a newline?
3. How to ignore feed that contains keywords that end with **apostrophe s**? (for example: Afghan's) also how to deal with the issue that there are several similar looking single quote characters! 
4. Can we use Regex? (so that all words that begin with Afghan upto next blank character or punctuation symbol can be filtered)?

**Comment by Alkarex** [comment]
1. It is not case sensitive, but there might be subtle differences depending on the actual database used (and collation parameters)
2. It is the same to use `OR` on the same line like you do or on multiple lines
3. Quotes must be escaped `\'`
4. No regex yet, but follow https://github.com/FreshRSS/FreshRSS/issues/3549 . And also, if you search for `abc`, it will also match `abcde` so your `intitle:'Afghan' OR intitle:'Afghanistan'` can be shortened to `intitle:'Afghan'`

See https://freshrss.github.io/FreshRSS/en/users/10_filter.html#with-the-search-field

**Reply by Alkarex** [reply]
For regex, tests welcome of https://github.com/FreshRSS/FreshRSS/pull/6706 @lamyergeier
